src/OpenAL/OpenToolkit.OpenAL/AL/AL.cs

```diff
@@ -354,9 +354,8 @@ public static int[] GenSources(int n)
         /// <returns>Pointer to an int value which will store the name of the new source.</returns>
         public static int GenSource()
         {
-            int temp;
-            GenSources(1, out temp);
-            return (int)temp;
+            GenSources(1, out int temp);
```

Comment:
I really am picky. I apologize. But can we rename `temp` into `source` or so?

Reply:
This is the nicest way I've seen someone nitpic.

Well done community!

Reply:
This is also fixed now